requirements/common.txt

```diff
@@ -1,2 +1,4 @@
 django==1.11.4
 psycopg2==2.7.3
+Pillow==4.2.1
+olefile==0.44
```

Comment:
Yep, I agree with that.
I've checked the [setup.py in Pillow](https://github.com/python-pillow/Pillow/blob/master/setup.py#L780).
It would be installed by `pip` once you install Pillow.